frontend/test/metabase/scenarios/question/timelines.cy.spec.js

```diff
@@ -8,18 +8,25 @@ describe("scenarios > collections > timelines", () => {
   describe("as admin", () => {
     beforeEach(() => {
       cy.signInAsAdmin();
+      cy.intercept("GET", "/api/collection/root").as("collections");
+      cy.intercept("POST", "/api/timeline-event").as("timelineEvent");
+      cy.intercept("PUT", "/api/timeline-event/**").as("putTimelineEvent");
```

Comment:
```
cy.intercept({
  method: '+(PUT|PATCH)',
  url: '/api/timeline-event/**',
}).as("timelineEvent");
```

This is how I understood from the docs. But maybe @nemanjaglumac can help better here.

Reply:
That doesn't really make sense because we might want to / need to distinguish between the two when we wait for them.

Reply:
But...

In case you ever need something like that, there's a bit simpler and easier solution. You can omit the method all together:
`cy.intercept("/api/timeline-event/**").as("timelineEvent");`